src/OrchardCore.Modules/OrchardCore.Flows/Controllers/AdminController.cs

```diff
@@ -112,12 +111,12 @@ private IEnumerable<ContentTypeDefinition> GetContainedContentTypes(string conte
 
             if (settings == null || settings.ContainedContentTypes == null || !settings.ContainedContentTypes.Any())
             {
-                return _contentDefinitionManager.ListTypeDefinitions().Where(t => t.GetSettings<ContentTypeSettings>().Stereotype == "Widget");
+                return _contentDefinitionManager.ListTypeDefinitions().Where(t => t.GetStereotypeOrDefault() == "Widget");
```

Comment:
What if it's default?!!

Reply:
`default(string) == “Widget”` will return false, so same result